packages/node/__tests__/index.test.js

```diff
@@ -96,180 +104,95 @@ describe('#metrics', () => {
     }).toThrow('You must provide a grouping function');
   });
 
-  it('should send a request to the metrics server', () => {
-    const apiMock = getReadMeApiMock(1);
-    const mock = nock(config.host, {
-      reqheaders: {
-        'Content-Type': 'application/json',
-        'User-Agent': `${pkg.name}/${pkg.version}`,
-      },
-    })
-      .post('/v1/request', ([body]) => {
-        expect(body.group).toBe(group);
-        expect(typeof body.request.log.entries[0].startedDateTime).toBe('string');
-        return true;
-      })
-      .basicAuth({ user: apiKey })
-      .reply(200);
+  it('should send a request to the metrics server', async done => {
```

Comment:
I'm still kind of waffling on how these tests are constructed. It would be nice if there was a no code way to construct these tests in a common config, but that would require the creation of a DSL, along with an implementation of that DSL in every language to run the tests. At that point we might as well just write the damn tests like I am here.